app/controllers/v1/projects_controller.rb

```diff
@@ -4,7 +4,8 @@ class ProjectsController < ApiController
 
 		def index 
 			category_params = params[:category]
-			projects = category_params == "" || !category_params ? Project.all : Project.by_category(category_params)
+			category_search = params[:search]
```

Comment:
This should be `seach_params` if you want to follow the above pattern for names.